snyk/models.py

```diff
@@ -72,6 +72,48 @@ class IssueSet(DataClassJSONMixin):
     issues: Issue
 
 
+@dataclass
+class IssueData(DataClassJSONMixin):
+    id: str
+    title: Optional[str] = None
+    severity: Optional[str] = None
+    originalSeverity: Optional[str] = None
+    url: Optional[str] = None
+    description: Optional[str] = None
+    identifiers: Optional[Any] = None
+    credit: Optional[str] = None
+    exploitMaturity: Optional[str] = None
+    semver: Optional[Any] = None
+    publicationTime: Optional[str] = None
+    disclosureTime: Optional[str] = None
+    CVSSv3: Optional[str] = None
+    cvssScore: Optional[str] = None
+    language: Optional[str] = None
+    patches: Optional[Any] = None
+    nearestFixedInVersion: Optional[str] = None
+
+
+@dataclass
+class AggregatedIssue(DataClassJSONMixin):
+    id: str
+    issueType: Optional[str] = None
+    pkgName: Optional[str] = None
+    pkgVersions: Optional[List[str]] = None
+    issueData: Optional[IssueData] = None
+    isPatched: Optional[bool] = None
+    isIgnored: Optional[bool] = None
+    fixInfo: Optional[Any] = None
+    # Not mentioned in schema but returned
+    # https://snyk.docs.apiary.io/#reference/projects/aggregated-project-issues/list-all-aggregated-issues
+    priorityScore: Optional[int] = None
+    priority: Optional[Any] = None
```

Comment:
The following classes list `id` as the only required key, please consult the [API documentation](https://snyk.docs.apiary.io/#reference/projects/aggregated-project-issues/list-all-aggregated-issues) for an additional  list of required and optional keys